### ISSUE TITLE
question of compute_accuracy

### ISSUE THREAD

**Issue by 18811449050** [QinbinLi/MOON#13]

**Comment by 18811449050** [comment]
I would like to know what this method(compute_accuracy) does? centralized learning？Model warm up？

**Comment by QinbinLi** [comment]
Hi @18811449050 ,

This method computes the prediction accuracy of the input model on the input dataset. There is no training on the model inside `compute_accuracy`.

**Comment by 18811449050** [comment]
`loss2 = mu * criterion(logits, labels)
 loss1 = criterion(out, target)
 loss = loss1 + loss2`

**Comment by QinbinLi** [comment]
Currently, MOON is designed for the classification task. For regression task, you may need to change the cross-entropy loss (e.g., line 130 of main.py) to loss for regression (e.g., MSELoss). You also need to change the model architecture.
